tests/e2e-playwright/helper/api.ts

```diff
@@ -0,0 +1,67 @@
+import { config } from '../../e2e/config.js'
+import { UsersEnvironment } from '../../e2e/support/environment'
+import { api } from '../../e2e/support'
+import { ResourceType } from '../../e2e/support/api/share/share'
+
+export async function createUserUsingAPI(
```

Comment:
and it might be better to follow consistent naming convention for step def functions
```suggestion
export async function userHasBeenCreated(
```